taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -740,6 +762,52 @@ export class VsCodeHelper {
 		);
 	}
 
+	exposeTestSetupCommand(projectDir?: Util.PathToDir) {
```

Comment:
The parameter `projectDir` is optional but the body of this function uses `projectDir` without checking if it's provided.

Reply:
Good point. Thank Jimmy.